bedrock/base/tests/test_views.py

```diff
@@ -38,3 +43,26 @@ def test_default_template(self):
     def test_no_country(self):
         template = self.get_template(None)
         assert template == "firefox-mobile.html"
+
+
+@pytest.mark.django_db
+def test_get_contentful_sync_info():
+
+    middle = tz_now()
+    first = middle - datetime.timedelta(hours=3)
+    last = middle + datetime.timedelta(hours=3)
+
+    for idx, timestamp in enumerate([middle, last, first]):
+        ContentfulEntry.objects.create(
+            contentful_id=f"id-{idx}",
+            last_modified=timestamp,
+        )
+
+    assert get_contentful_sync_info() == {
+        "latest_sync": last,
```

Comment:
shouldn't this fail now since `time_since_latest_sync` should also be in the dict?

Reply:
I think it is failing in CI